Packs/AttachmentsPreProcessing/Scripts/CreateFileFromPathObject/CreateFileFromPathObject.py

```diff
@@ -0,0 +1,9 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+obj = json.loads(demisto.args()["object"])
+for file in obj:
```

Comment:
Using 'file' as a parameter is a bad practice as 'file' is a special saved word.